src/ucp/core/ucp_request.h

```diff
@@ -111,8 +109,13 @@ enum {
  * Request in progress.
  */
 struct ucp_request {
-    ucs_status_t                  status;     /* Operation status */
-    uint32_t                      flags;      /* Request flags */
+    /* Operation status */
+    ucs_status_t      status;
+    /* Request flags */
+    uint32_t          flags;
+    /* Local request ID taken from PTR MAP */
+    ucs_ptr_map_key_t local_req_id;
```

Comment:
pls rename to `id`

Reply:
I guess it is more clear to have `local_req_id` name, because we have `remote_req_id`

Reply:
i think req->id is very simple and clear it's the local